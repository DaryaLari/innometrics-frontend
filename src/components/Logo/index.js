import React from 'react'
import styles from './style.css'

class Logo extends React.Component {
  render() {
    return (
        <div className={styles.logo}>
          <span className={styles.name}>
            <span className={styles.inno}>Inno</span>
            <span className={styles.metrics}>Metrics</span>
          </span>
        </div>

    )
  }
}

export default Logo