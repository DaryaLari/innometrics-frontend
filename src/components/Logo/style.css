.logo {
    font-size: x-large;
}

.name {
    letter-spacing: -1px;
}

.inno {
    color: rgba(250, 150, 50, 1);
}

.metrics {
    color: rgba(25, 255, 255, 1);
}
