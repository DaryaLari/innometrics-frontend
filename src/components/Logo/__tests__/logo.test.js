import React from 'react'
import TestRenderer from 'react-test-renderer';
import Logo from '../index'

describe('Test Logo component', () => {

  it('Test render Logo component', () => {

    const instance = TestRenderer.create(
      <Logo/>
    )
    expect(instance.toJSON()).toMatchSnapshot()
  })
})