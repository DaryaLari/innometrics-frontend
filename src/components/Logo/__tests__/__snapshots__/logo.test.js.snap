// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Logo component Test render Logo component 1`] = `
<div>
  <span>
    <span>
      Inno
    </span>
    <span>
      Metrics
    </span>
  </span>
</div>
`;
